curriculum/challenges/_meta/quality-assurance-and-testing-with-chai/meta.json

```
@@ -9,26 +9,11 @@
   "superBlock": "quality-assurance",
   "superOrder": 6,
   "challengeOrder": [
-    [
-      "587d824a367417b2b2512c46",
-      "Learn How JavaScript Assertions Work"
-    ],
-    [
-      "587d824b367417b2b2512c47",
-      "Test if a Variable or Function is Defined"
-    ],
-    [
-      "587d824b367417b2b2512c48",
-      "Use Assert.isOK and Assert.isNotOK"
-    ],
-    [
-      "587d824b367417b2b2512c49",
-      "Test for Truthiness"
-    ],
-    [
-      "587d824b367417b2b2512c4a",
-      "Use the Double Equals to Assert Equality"
-    ],
+    ["587d824a367417b2b2512c46", "Learn How JavaScript Assertions Work"],
```

```suggestion
    [
      "587d824a367417b2b2512c46",
      "Learn How JavaScript Assertions Work"
    ],
```
These could probably all be spaced out for consistency. Probably easier to read on one line like that, but I think all the meta files have them spaced out.

---

I agree, but it was the recommended fCC prettier settings that did that (line too long). So, I am happy to revert it, but should I just add the file to a `.prettierignore` file?